pkgs/development/tools/vend/default.nix

```diff
@@ -1,28 +1,36 @@
 { stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoModule {
+buildGoModule rec {
   pname = "vend";
-  version = "unstable-2020-06-04";
 
-  patches = [./remove_tidy.patch];
+  /*
+  This package is used to generate vendor folders for
+  packages that use the `runVend` option with `buildGoModule`.
+
+  Do not update this package without checking that the vendorSha256
+  hashes of packages using the `runVend` option are unchanged
+  or updating their vendorSha256 hashes if necessary.
+  */
+  version = "1.0.2";
+  # Disable the bot
+  # nixpkgs-update: no auto update
```

Comment:
I assume we can keep have the bot still try updating - if the patch applies cleanly. If not, manual intervention is needed anyways.

Reply:
I'm assuming that the comment would be overlooked if the bot sends a PR.

Reply:
the bot won't update `vendorSha256`?

Reply:
> the bot won't update vendorSha256?

I don't understand?

Reply:
Updates needs to be very carefully reviewed, i.e. commit-by-commit. Nixpkgs-review won't find breakages with fixed-input derivations. We don't want people to merge those changes if they don't understand the implications. They might do so if they see updates made by @r-ryantm 
In fact I would prefer having not to update this package unless we need the update.

Reply:
> the bot won't update `vendorSha256`?

It would only update the checksum of this package but not check for all `vendorSha256` in the repo.

Reply:
Right, makes sense.